backend/views/syson-tree-explorer-view/src/main/java/org/eclipse/syson/tree/explorer/view/SysONExplorerTreeDescriptionOrderer.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.tree.explorer.view;
+
+import java.util.List;
+
+import org.eclipse.sirius.web.application.views.explorer.dto.ExplorerDescriptionMetadata;
+import org.eclipse.sirius.web.application.views.explorer.services.ExplorerDescriptionProvider;
+import org.eclipse.sirius.web.application.views.explorer.services.api.IExplorerTreeDescriptionOrderer;
+import org.springframework.stereotype.Service;
+
+/**
+ * Implementation used to sort tree descriptions that can be used as explorer for Domain, by placing the default
```

Comment:
Please update this comment, it seems it has been copied from elsewhere

Reply:
done